test-packages/e2e-tests/test/mail/mail.spec.ts

```diff
@@ -47,37 +50,48 @@ describe('Mail', () => {
 
 async function sendTestMail(
   connection?: net.Socket,
-  ...mailOptions: Mail.Options[]
-): Promise<void> {
-  // create reusable transporter object using the default SMTP transport
-  const transporter = nodemailer.createTransport({
-    connection,
-    host: 'localhost',
-    port: 5566,
-    // true for 465, false for other ports
-    secure: false,
-    auth: {
-      user: 'user',
-      pass: 'pd'
-    },
-    tls: {
-      // disable tls config to fix the self signed certificate error
-      rejectUnauthorized: false
-    }
-  });
-
-  for (const mailOptionIndex in mailOptions) {
-    // eslint-disable-next-line no-console
-    console.log(`Sending email ${mailOptionIndex}/${mailOptions.length}...`);
-    const response = await transporter.sendMail(mailOptions[mailOptionIndex]);
-    // eslint-disable-next-line no-console
-    console.log(
-      `...email ${mailOptionIndex}/${mailOptions.length} for subject "${mailOptions[mailOptionIndex].subject}" was sent successfully.`
-    );
-  }
-  transporter.close();
-  // eslint-disable-next-line no-console
-  console.log('SMTP transport connection closed.');
+  ...mailOptions: MailOptions[]
```

Comment:
[q] where is the local mail server setup and started? I do not see a before and after step in this test?

Reply:
Here it is:
https://github.com/SAP/cloud-sdk-js/blob/main/test-packages/e2e-tests/package.json#L10